src/main/java/frc/robot/RobotContainer.java

```diff
@@ -162,21 +165,47 @@ public void initializeAutoBuilder() {
   }
 
   public void registerCommands() {
-    NamedCommands.registerCommand("Intake_Note_1.5_Sec", new IntakeNote(intake, index, arm).withTimeout(1)); // 3 seconds used at SVR
+    NamedCommands.registerCommand("Intake_Note_1.5_Sec", new IntakeNote(intake, index).withTimeout(1)); // 3 seconds used at SVR
+    NamedCommands.registerCommand("Stop", new RunCommand(()->drive.stop()).withTimeout(1)); 
```

Comment:
Add drive as a second parameter after the lambda expression to make this work (currently it doesn't require the drivetrain). Also, add a check for any subsystems that are null so this won't crash on other robots.